charter.md

```diff
@@ -53,15 +53,245 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+are replacing, regardless of the length of that remainder.
+
+[voter-eligibility]: /elections.md#eligibility-for-voting
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Continuity
+
+If the committee is dissolved completely by any mechanism, selected members
+(selection described below) agree to be "continuity advisors" and attend
+meetings of the subsequently elected incoming committee in a non-voting
+capacity for 6 meetings or until the incoming committee dismisses them,
+whichever is first.
+
+##### Selection
+
+At least two (2) and not more than three (3) members shall be selected for
+this duty.
+
+* One shall be the longest-serving outgoing committee member. If multiple, the
+  incoming committee may select between them.
+* The second shall be selected by the incoming committee at their discretion.
+* The incoming committee, at their discretion, may select a third.
+
+#### Dissolution by vote
+
+Dissolution requires an affirmative vote of a **_two-thirds supermajority of
+the [fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* An affirmative vote of 5 members would be required to dissolve the committee
+
+The committee must specify a date of dissolution.
+If the committee is dissolved, all seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+If at any time a **_majority of the
+[fixed membership of the committee](#composition)_**'s seats are vacant, the entire committee will be dissolved.
+
+Example:
+
+* 7 (seats) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* If 4 committee seats are vacant, the entire committee will dissolve
+
+The number of vacant seats will be evaluated prior to any replacement of
+previously-seated members.
+
+### Removal
+
+A committee member may be removed from the committee by the other committee
+members. The member should be given the opportunity to resign before they are
+removed.
+
+Removal should be considered for the following reasons:
+
+* The member has been found to have committed a code of conduct violation.
+* The member is convicted of a felony.
+* The member has been completely out of contact for more than 30 consecutive
+  calendar days without having made prior arrangements.
+* The member has explicitly, publicly violated the privacy of a community
```

Comment:
This is weirdly specific, how would this not be a CoC violation? I would remove this.

Reply:
@lavalamp guessing this is outside of community spaces?

Reply:
(then CoC can't step in)

Reply:
@dims maybe? It's not what it says, if that's what it means it should be clear? (and then why not other aspects of the CoC also?)

If we want an extra-special CoC for board members, I don't think that's something that can be worked out in a short amount of time while people have an example in mind.

Reply:
agree, https://github.com/kubernetes/steering/pull/248#discussion_r915034333 we want to just delete this list